### PR TITLE
Schema cleanup/refactor

### PR DESCRIPTION
**Pull request by bbrands02**

### CODE REVIEW

File: Installation/Mapping/woo.xxllncCaseToWoo.mapping.json [hunk 1]
```diff
@@ -2,46 +2,43 @@
...
-    "Bijlagen": "[{% set index=0 %}{% if values['attribute&#46;woo_publicatie']|default %}{% for document in values['attribute&#46;woo_publicatie'] %}{% if index != 0 %},{% endif %}{{ map('https://commongateway.nl/mapping/pdd.xxllncDocumentSetSourceId.schema.json', document)|json_encode }}{% set index=index+1 %}{% endfor %}{% endif %}]",
-    "Themas.0.Hoofdthema": "{% if values['attribute&#46;woo_thema']|default %}{{ values['attribute&#46;woo_thema'] }}{% endif %}"
+    "bijlagen": "[{% set index=0 %}{% if values['attribute&#46;woo_publicatie']|default %}{% for document in values['attribute&#46;woo_publicatie'] %}{% if index != 0 %},{% endif %}{{ map('https://commongateway.nl/mapping/pdd.xxllncDocumentSetSourceId.schema.json', document)|json_encode }}{% set index=index+1 %}{% endfor %}{% endif %}]",
+    "themas.0.hoofdthema": "{% if values['attribute&#46;woo_thema']|default %}{{ values['attribute&#46;woo_thema'] }}{% endif %}"
   },
   "cast": {
-    "Themas.0.Hoofdthema": "unsetIfValue==",
-    "Themas.0": "unsetIfValue==",
+    "themas.0.Hoofdthema": "unsetIfValue==",
```
Comment (rjzondervan) [inline Installation/Mapping/woo.xxllncCaseToWoo.mapping.json:28]: Shouldn't this be themas.0.hoofdthema

**Review by rjzondervan — APPROVED** [review]

**Review by rjzondervan — CHANGES_REQUESTED** [review]
Found a renamed object without renaming the cast

**Review by rjzondervan — APPROVED** [review]

**Review by rjzondervan — APPROVED** [review]

File: Installation/Mapping/woo.xxllncDocumentToBijlage.mapping.json [hunk 2]
```diff
@@ -2,13 +2,14 @@
...
-    "Titel_Bijlage": "filename",
-    "URL_Bijlage": "url",
-    "Tijdstip_laatste_wijziging_bijlage": "{{ \"now\"|date(\"Y-m-d H:i:s\") }}",
-    "Status_Bijlage": "{% if accepted|default and accepted == 1 %}{{ \"accepted\" }}{% endif %}"
+    "titel": "filename",
+    "label": "filename",
+    "url": "url",
+    "tijdstip_laatste_wijziging": "{{ \"now\"|date(\"Y-m-d H:i:s\") }}",
+    "status": "{% if accepted|default and accepted == 1 %}{{ \"accepted\" }}{% endif %}"
   }
```
Comment (WilcoLouwerse) [inline Installation/Mapping/woo.xxllncDocumentToBijlage.mapping.json:14]: Just checking, does this mapping not need any casts?

File: Installation/Data/WooTestdata.json [hunk 3]
```diff
@@ -1,355 +1,357 @@
...
     {
-      "UUID": "8c713806-6372-4630-95f4-a9021b2a41c9",
-      "ID": "2023-002106",
-      "Object_ID": 3950,
-      "Portal_url": "https://www.barendrecht.nl/openwoo/werkprocedure-aanleg-glasvezelnetwerk/8c713806-6372-4630-95f4-a9021b2a41c9",
-      "Titel": "Werkprocedure aanleg glasvezelnetwerk",
-      "Samenvatting": "Dit verzoek is een vervolg op het Woo-besluit van 8 juni 2023 waarbij informatie openbaar is gemaakt over de voorbereiding en plaatsing van een schakelkast glasvezelnetwerk. In dit verzoek is met een beroep op de Wet open overheid om informatie gevraagd over de werkprocedure voor de aanleg van glasvezelnetwerk en dan met name de plaatsing van een schakelkast glasvezelnetwerk.",
-      "Ontvangstdatum": "2023-06-14",
-      "Besluitdatum": "2023-07-11",
+      "id": "2023-002106",
```
Comment (WilcoLouwerse) [inline Installation/Data/WooTestdata.json:4]: Have you also tested this locally, this test data?
Does this work with the "id" field?
Doesn't the gateway try to set the ObjectEntity id with this value (which should be a uuid)?

**Review by WilcoLouwerse — APPROVED** [review]

File: Installation/Mapping/woo.xxllncDocumentToBijlage.mapping.json (hunk 2 above)
Comment (bbrands02) [reply Installation/Mapping/woo.xxllncDocumentToBijlage.mapping.json:14]: no :)

**Pull request merged**